kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -140,7 +140,7 @@ public void testGetSolverState() throws Exception {
         assertEquals( SOLVER_1_ID, returnedInstance.getSolverId() );
         assertEquals( SolverInstance.getSolverInstanceKey( CONTAINER_1_ID, SOLVER_1_ID ), returnedInstance.getSolverInstanceKey());
         assertEquals( SolverInstance.SolverStatus.NOT_SOLVING, returnedInstance.getStatus());
-        assertNull( returnedInstance.getScore() );
+        assertNull( returnedInstance.getScoreWrapper() );
```

Comment:
Looking at it from a client perspective, I 'd argue that the ScoreWrapper instance should be null if and only if the method I just called doesn't fill it in because it's not part of its contract (think javadoc). If it did fill it in, but the score is null because the solver hasn't assigned it yet, then the ScoreWrapper should be empty.

Why? Think of this code:
response = clientApi.callService(...);
Score score = response.getSolverInstance().getScoreWrapper().toScore(); // NPE otherwise!
if (score == null) {
   print("No score assigned yet");
}